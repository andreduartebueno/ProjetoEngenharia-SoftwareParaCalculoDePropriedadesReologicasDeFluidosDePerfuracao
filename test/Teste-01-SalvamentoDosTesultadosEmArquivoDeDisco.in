#Teste Interface Software
#Teste 1:  Salvamento dos resultados em arquivo de disco

#Dados de entrada
Coloque aqui os dados de entrada


