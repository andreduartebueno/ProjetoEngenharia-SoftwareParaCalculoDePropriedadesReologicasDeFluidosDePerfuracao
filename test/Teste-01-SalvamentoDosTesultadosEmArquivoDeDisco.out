#Teste Interface Software
#Teste 1:  Salvamento dos resultados em arquivo de disco

Robert, execute o teste;
No terminal selecione o texto todo do teste e cole aqui.
De forma que aqui teremos toda a saída do programa.

Faça o mesmo para todos os testes;
Adicionar dados de entrada no arquivo Teste-x-NomeTeste.in;
Adicionar dados de saida no  arquivo Teste-x-NomeTeste.out

Os nomes dos testes devem estar de acordo com o que tem no lyx.
Assim a pessoa acha rapidinho.


